Makefile

```diff
@@ -56,6 +58,14 @@ build.docker: $(BIN_FILENAME) ## Build the docker image
 clean: ## Cleans up the generated resources
 	rm -rf dist/ cover.out $(BIN_FILENAME) || true
 
+.PHONY: local-env
+local-env-setup: ## Setup local kind-based dev environment
+	$(localenv_make) setup
+
+.PHONY: local-env-clean-setup
+local-env-clean-setup: ## Setup local kind-based dev environment
```

Comment:
The comment is wrong and IMO the tag misleading. I would just call it `local-env-clean`